third_party/LICENSES_BUNDLED.txt

```diff
@@ -0,0 +1,264 @@
+The Pytorch repository and source distributions bundle several libraries that are 
+compatibly licensed.  We list these here.
+
+Name: FP16
+License: MIT License
+Files: third_party/FP16
+  For details, see third_party/FP16/LICENSE
+
+Name: FXdiv
+License: MIT License
+Files: third_party/FXdiv
+  For details, see third_party/FXdiv/LICENSE
+
+Name: NNPACK
+License: BSD-2-Clause License
+Files: third_party/NNPACK
+  For details, see third_party/NNPACK/LICENSE
+
+Name: QNNPACK
+License: BSD-3-Clause License
+Files: third_party/QNNPACK
+  For details, see third_party/QNNPACK/LICENSE
+
+Name: XNNPACK
+License: BSD-3-Clause License
+Files: third_party/XNNPACK
+  For details, see third_party/XNNPACK/LICENSE
+
+Name: benchmark
+License: Apache License v2.0
```

Comment:
Use SPDX identifiers?
```suggestion
License: Apache-2.0
```

Reply:
Thanks. That led me to [this list of identifiers](https://spdx.org/licenses). Maybe the bsd variants I found are valid? I will have to investigate.

Reply:
Searching with spdx led me to https://github.com/spdx/tools-python so I will try using that instead of my simple text matching

Reply:
The only way to really use spdx software is via java, and it is too complicated. I am staying with my simple parser for now. Fixed the SPDX identifier.